### ISSUE TITLE
Deprecation of #deliver on Rails 4.2.0

### ISSUE THREAD

**Issue by everton** [smartinez87/exception_notification#249]
Running on 4.2.0 I found this warning:

```
DEPRECATION WARNING: `#deliver` is deprecated and will be removed in Rails 5. Use `#deliver_now` to deliver immediately or `#deliver_later` to deliver through Active Job.
```

There is any plan to update this? It will be just a replace of #deliver with #deliver_now or it will require more logic around this to deal with background emails? Any way I can help with it?

**Comment by dsilver829** [comment]
+1

**Comment by jaredbeck** [comment]
Is there a release with this fix?  It looks like the last release was 4.1.0.rc1 on April 16, 2014, so I guess not.  @smartinez87 can you cut a new release please?  Another release candidate (4.1.0.rc2) would be fine by me.  Thanks!

**Comment by jaredbeck** [comment]
> 4.1.0.rc2 - March 25, 2015
> https://rubygems.org/gems/exception_notification/versions/4.1.0.rc2

Thanks for the quick release, Sebastian!
